app/scripts/filters/resources.js

```diff
@@ -1084,7 +1084,18 @@ angular.module('openshiftConsole')
       return lastFinishTime;
     };
   })
+  // gets the first status condition.
   .filter('statusCondition', function() {
```

Comment:
Ill do a search again, but I thought I updated the other refrences. Let me check.